packages/Genprint/Genprint.0.1/opam

```diff
@@ -0,0 +1,24 @@
+opam-version: "2.0"
+name: "Genprint"
+version: "0.1"
+synopsis: "PPX syntax extension and library package for printing values of any type"
+description: """
+A PPX syntax extension and library package enabling printing of values of any type using OCaml's internal printing facilities a la toplevel evaluation.
+Useful for debugging as a quick alternative to ocamldebug/ppx_deriving/#install_printer.
+"""
+
+maintainer: "progman1"
+authors: [ "progman1" ]
+homepage: "github.com/progman1/genprintlib"
+dev-repo: "git+https://github.com/progman1/genprintlib.git"
+bug-reports: "progman1"
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" "genprint"]
```

Comment:
```suggestion
  ["dune" "build" "-p" "genprint" "-j" jobs]
```
This adds people to choose how many cores they want to give to compile your package